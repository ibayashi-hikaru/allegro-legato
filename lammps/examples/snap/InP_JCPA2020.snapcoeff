../../potentials/InP_JCPA2020.snapcoeff