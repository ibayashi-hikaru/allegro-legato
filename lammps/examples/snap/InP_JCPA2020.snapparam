../../potentials/InP_JCPA2020.snapparam