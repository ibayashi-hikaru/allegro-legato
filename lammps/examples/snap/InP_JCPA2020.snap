../../potentials/InP_JCPA2020.snap